solutions/EMFSolutionAOF/.project

```diff
@@ -5,6 +5,11 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.xtext.ui.shared.xtextBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
```

Comment:
Are these changes to the EMFSolutionAOF .project file intentional? Should we keep them?

Reply:
No, they are not. Please ignore them.